synchronizer/metrics/metrics.go

```diff
@@ -21,7 +21,7 @@ const (
 	FullL1SyncTimeName = Prefix + "full_L1_sync_time"
 
 	// FullSyncIterationTimeName is the name of the label for a L1 synchronization.
-	FullSyncIterationTimeName = Prefix + "full_L1_sync_time"
```

Comment:
This name conflicts with the above histogram name, so this metric isn't being tracked currently.